Automattic/vip-go-mu-plugins
# Debug bar panels: Fix Warning for Query Summary.

## Description

Defensive coding to stop the warnings:

```
Warning: array_key_exists() expects parameter 2 to be array, null given in /var/www/wp-content/mu-plugins/vip-helpers/vip-debug-bar-panels.php on line 132
```

```
Warning: array_key_exists() expects parameter 2 to be array, null given in /var/www/wp-content/mu-plugins/vip-helpers/vip-debug-bar-panels.php on line 157
```

<img width="1193" alt="Screenshot 2019-11-29 at 11 54 29" src="https://user-images.githubusercontent.com/88371/69867750-a543b780-129f-11ea-98b1-026c39aee72b.png">

Possibly only happens when a user has not completed 2FA.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Log in to a site that requires 2FA, but your account does not.
2. Check Debug Bar -> Query Summary to see the above screenshot.
3. Check logs to see Warning.
4. Apply PR.
1. Reload and check warning is gone.
